src/player_event_handler.rs

```diff
@@ -0,0 +1,50 @@
+use std::process::Command;
+use std::sync::mpsc::{channel, Sender};
+use std::thread;
+use librespot::playback::config::PlayerEvent;
+
+fn run_program(program: &str, args: Vec<String>) {
+    info!("Running {}", program);
```

Comment:
Include the arguments in here. Also may be better to print the split version of program, as it may help debugging eg. someone expecting librespot to properly handle quotes/escaped whitespaces.

As a side note, we'd want something like [shellwords](https://docs.rs/shellwords) for smarter splitting, but this does not have to be in this PR.